pkg/lint/lintersdb/validator.go

```diff
@@ -17,19 +17,19 @@ func NewValidator(m *Manager) *Validator {
 	}
 }
 
-func (v Validator) validateLintersNames(cfg *config.Linters) error {
+func (v *Validator) validateLinterConfigNames(cfg *config.Linters) error {
```

Comment:
agreed that changing to pointer is not related to the scope of this PR.

I don't think it's required as well.

Reply:
ok, fixed.